packages/components/src/components/chat/Chat.vue

```diff
@@ -45,15 +45,15 @@
     </div>
     <div v-if="!authorized" class="status-unauthorized status-container">
       <div class="status-label">
-        <p>You must be logged in to AppMap to use this feature.</p>
+        <p>You must have activated AppMap to use this feature.</p>
         <p>
           <b>VSCode</b>
-          To login with VSCode, run the command <tt>AppMap: Login</tt>, or click the Sign In link
-          from the AppMap panel.
+          To activate AppMap for VSCode, run the command <tt>AppMap: Login</tt>, or activate using
```

Comment:
Hmm, what does AppMap: Login do? Maybe we can just point them to the sidebar, or is there a command to show/reveal the sidebar?